src/zenml/integrations/tensorflow/materializers/keras_materializer.py

```diff
@@ -18,8 +18,8 @@
 
 from zenml.artifacts import ModelArtifact
 from zenml.io import fileio
-from zenml.io import utils as fileio_utils
 from zenml.materializers.base_materializer import BaseMaterializer
+from zenml.utils import io_utils as fileio_utils
```

Comment:
We can now use `io_utils` here for consistency.

Reply:
Done. (And caught this in a bunch of other places as well.)